api/db/populate-messages.js

```diff
@@ -0,0 +1,55 @@
+const { getArticle } = require('./utils');
+
+const Friendship = require('../models/Friendship');
+const Message = require('../models/Message');
+
+let displayLogs;
+
+const createMessage = async (messageParams) => {
+  const message = new Message({
+    friendship: messageParams.friendship,
+    user: messageParams.user,
+    content: messageParams.content,
+  });
+  message.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New message created: ${message}`);
+  }
+  return message;
+};
+
+const createMessages = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating messages collection...');
+  displayLogs = verbose;
+
+  const friendshipsArray = await Friendship.find();
+  const messagesArray = [];
+
+  for (const friendship of friendshipsArray) {
+    const messagesCount = Math.floor(Math.random() * 10);
+
+    for (let i = 0; i < messagesCount; i++) {
+      const user =
+        Math.random() < 0.5 ? friendship.requestant : friendship.receivant;
+      const content = getArticle();
```

Comment:
As said above, I think a real conversation would consist mostly of smaller messages, so maybe we can throw a random number here to decide if we call the `getArticle()` function (should be improbable) or generate just a sentence. How does that sound?